kustomize/README.md

```diff
@@ -78,6 +78,8 @@ Required Operators:
 
 - Cluster Monitoring
 - Bitnami sealed secrets (via controller deployment below)
+- Openshift Serverless is installed with [this guide](https://docs.openshift.com/container-platform/4.8/serverless/install/install-serverless-operator.html).
```

Comment:
are we using 4.8? or a later one, at this point?

Reply:
at the moment our `dev` cluster is using `4.8` and `stable` is using `4.9`